src/test/java/core/basesyntax/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String INPUT_FILE_PATH
+            = "src/test/java/core/basesyntax/resources/testexample.csv";
+    private static final String NO_FILE = "";
+    private final FileReaderService readerService = new FileReaderServiceImpl();
```

Comment:
initialize services in methods annotated Before or BeforeClass (depends on the version of Junit), fix each test accordingly